samples/boards/nrf52/mesh/onoff_level_app/README.rst

```diff
@@ -0,0 +1,106 @@
+.. _bluetooth-mesh-onoff-level-sample:
+Bluetooth: Mesh OnOff & Level Model
```

Comment:
Got rid of too many blank lines :)  
Put a blank line before this document title.